RELEASE.md

```diff
@@ -1,88 +1,34 @@
-
-The GitHub binary releases of the roles are handled in the `release-bin.yaml` workflow.
-
-This workflow is manually started by navigating to the "Actions" tab in the SRI repo, then navigating to the Release workflow section, and clicking "Run Workflow".
-
-Every time the workflow is manually triggered, the correct release branch must be chosen.
-
-Note: in order to be able to manually trigger the "Run Workflow" button, the user needs to have "Write" permissions on the repository, otherwise the button will not show up on the UI.
-
-# Publishing Library Crates
-
-Although SRI has a global release cycle, which is attached to the binaries, each internal crate also has its own versioning history.
-
-Lib crates are published to crates.io in the `release-lib.yaml` workflow. The workflow tries to update all the library crates. 
-
-If a crate is not updated successfully, the step will fail, but since all steps have `continue-on-error` set to true, the workflow will continue.
-
-Since steps can fail, the output of the action must be manually checked.
-
-Every PR to `main` needs to increase the version of whatever crate it is touching. Otherwise, we will mess up the dependency chain of whoever is fetching from crates.io
-
-Manually running `cargo release` in the various workspaces helps to prepare the version number before the releases are published.
-
-Every time we bump some crate's version, `release-libs.yaml` needs to be manually triggered in order to update crates.io.
+## Release Process
+
+- Constrain each development cycle to a fixed time period, after which a release
+  is made. Typically, the development cycle is 6-8 weeks long.
+- The release process is initiated by creating a new release branch from the
+  `main` branch. The release branch is named `x.y.z`, where `x.y.z` is the
+  version number of the release.
+- The release branch is used as a breaking point (and future reference) for the
+  next release and we leave the `main` branch active at all times.
+- If any bugs are found in the release branch, they should be fixed in the
+  release branch.
+- Create a Pull Request from release branch to the `main` branch.
+- The release branch is then tagged(with release notes) with the version number
+  of the release.
+- Release workspace crates to crates.io.
+
+## Versioning
+
+Protocol and Utils crates follow SemVer 2.0.0. The version number is stored in
+the `Cargo.toml` file of each crate. If a breaking change is introduced to one
+of the crates, the version number must be updated accordingly, otherwise a
+SemVer CI check would fail. Note that this does not apply to the `roles` and
+other crates in the repository.
+
+The general SRI version does not follow any SemVer rules. You can track the
+releases on Github Releases in the repository.
+
+## Tags and Branches
+
+- The `main` branch is the default branch and it is always active.
+- The `main` branch is protected and requires a pull request to merge changes
```

Comment:
should we add some words here about all PRs being merged to `main` via merge commits, and not rebases or squashes?

cc @GitGab19

Reply:
Yeah, we can add a sentence like:
"Changes to `main` branch are added through a merge commit"

Reply:
added